targets/TARGET_STM/mbed_overrides.c

```diff
@@ -292,7 +292,11 @@ void mbed_sdk_init()
 #if (MBED_CONF_TARGET_RTC_CLOCK_SOURCE == USE_RTC_CLK_HSE)
     RCC_PeriphCLKInitTypeDef PeriphClkInitStruct = {0};
     PeriphClkInitStruct.PeriphClockSelection = RCC_PERIPHCLK_RTC;
+#if (!TARGET_STM32F1)
```

Comment:
What about:
`#if defined(RCC_RTCCLKSOURCE_HSE_DIVX)`

Reply:
> What about:
> `#if defined(RCC_RTCCLKSOURCE_HSE_DIVX)`

This should work as well. I will change it asap.